scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/clazz/ClassDefinitionDiscovery.scala

```diff
@@ -32,6 +32,7 @@ class ClassDefinitionDiscovery(classDefinitionExtractor: ClassDefinitionExtracto
   private val mandatoryClasses = (Set(
     classOf[java.util.List[_]],
     classOf[java.util.Map[_, _]],
+    classOf[java.util.Map.Entry[_, _]],
```

Comment:
Interesting that we had examples for that, but it wasn't possible to use it :)

Reply:
afaik it was possible because it was artificially handled in `Typer.extractIterativeType`, but started to fail tests when I added runtime checking in property accessors.